backend/test/integration/chatController.test.ts

```diff
@@ -42,50 +47,50 @@ function responseMock() {
 	} as unknown as Response;
 }
 
-describe('handleChatToGPT integration tests', () => {
-	const testSentEmail: EmailInfo = {
-		address: 'bob@example.com',
-		body: 'Test body',
-		subject: 'Test subject',
+function chatResponseAssistant(content: string) {
+	return {
+		choices: [
+			{
+				message: {
+					role: 'assistant',
+					content,
+				},
+			},
+		],
 	};
+}
 
-	function chatResponseAssistant(content: string) {
-		return {
-			choices: [
-				{
-					message: {
-						role: 'assistant',
-						content,
-					},
-				},
-			],
-		};
-	}
+const testSentEmail: EmailInfo = {
+	address: 'bob@example.com',
+	body: 'Test body',
+	subject: 'Test subject',
+};
 
-	function chatSendEmailResponseAssistant() {
-		return {
-			choices: [
-				{
-					message: {
-						tool_calls: [
-							{
-								type: 'function',
-								id: 'sendEmail',
-								function: {
-									name: 'sendEmail',
-									arguments: JSON.stringify({
-										...testSentEmail,
-										confirmed: true,
-									}),
-								},
+function chatSendEmailResponseAssistant() {
+	return {
+		choices: [
+			{
+				message: {
+					tool_calls: [
+						{
+							type: 'function',
+							id: 'sendEmail',
+							function: {
+								name: 'sendEmail',
+								arguments: JSON.stringify({
+									...testSentEmail,
```

Comment:
I've had to move `chatSendEmailResponseAssistant`, `chatResponseAssistant` and `chatSendEmailResponseAssistant` so that I can use them to define the type of `mockCreateChatCompletion` (line 26). The other option would be to keep them where they are and write a manual type definition, but then you run the risk of them coming out of sync...
Neither option seems particularly pretty.